src/NosCore.Handler/CommandPacketHandler.cs

```diff
@@ -31,11 +31,13 @@ public void Speed(SpeedPacket speedPacket)
         {
```

Comment:
That's a bit heavy, why not using ternary expression instead of duplicating code ? 

Session.Character.Speed = (speedPacket.Speed >= 60 ? 59 : speedPacket.Speed);